edit/csslint-ruleset.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/**
+ * CSSLint Ruleset values
+ * 0 = disabled; 1 = warning; 2 = error
+ */
+window.csslintDefaultRuleset = {
```

Comment:
Why `ruleset`? Can't we use `rules` for all linters? If we can then all occurrences of "ruleset" should be replaced.

Reply:
That's just what CSSLint calls them... rules (the actual definitions) are separate from ruleset (the state of the rule: disabled, warning or error).

Reply:
Ah, I see the reason, but for Stylus those are identical entities - linter config. Maybe `config` or `ruleConfig` etc.